src/STEMMUS_SCOPE_exe.m

```diff
@@ -29,6 +29,7 @@ function STEMMUS_SCOPE_exe(config_file, runMode)
                        'GroundwaterSettings' ... % groundwater settings including input data from MODFLOW
                        'gwfluxes' ... % structure that includes groundwater recharge and its individual components
                        'EVAP' ... % evaporation
+                       'Trap' ... % transpiration

```

Comment:
why "Trap" is added here?

Reply:
For adding variables to BMI, two files should be changed, the `STEMMUS_SCOPE_exe.m` in this repo and `varaible_refence.py` in pystemmusscope. This is beyond the scope of issue #241 and should be done in a separate pull request. You can also add a comment about it in issue https://github.com/EcoExtreML/STEMMUS_SCOPE/issues/209.